dataactcore/aws/s3UrlHandler.py

```diff
@@ -132,3 +132,14 @@ def get_file_urls(self, bucket_name, path):
                 urls[file_name] = url
 
         return urls
+
+    @staticmethod
```

Comment:
Do we want to do that in this PR (there are other PRs this will almost definitely break) or do we want to do it first thing in the next sprint?